catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/GmdConverter.java

```diff
@@ -139,102 +141,207 @@ protected XmlTree buildTree(Set<Path> paths) {
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.CONTACT_PHONE_PATH,
+                Contact.POINT_OF_CONTACT_PHONE);
+
+        addListAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.CONTACT_ADDRESS_DELIVERY_POINT_PATH,
+                Contact.POINT_OF_CONTACT_ADDRESS);
+
+        addDateAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.DATE_TIME_STAMP_PATH,
+                Core.METACARD_MODIFIED);
 
-        pathValueTracker.add(new Path(GmdConstants.DATE_TIME_STAMP_PATH),
-                XSD_FACTORY.newXMLGregorianCalendar(modifiedCal)
-                        .toXMLFormat());
+        addCRSInformation(metacard, pathValueTracker);
 
         addIdentificationInfo(metacard, pathValueTracker);
 
         addDistributionInfo(metacard, pathValueTracker);
 
         return pathValueTracker;
-
     }
 
     protected void addDistributionInfo(MetacardImpl metacard,
             XstreamPathValueTracker pathValueTracker) {
+        addStringAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.FORMAT_PATH,
+                Media.FORMAT,
+                null);
+
+        addStringAttributeToXml(metacard,
+                pathValueTracker,
+                GmdConstants.FORMAT_VERSION_PATH,
+                Media.FORMAT_VERSION,
+                null);
 
         String resourceUrl = null;
-        Attribute downloadUrlAttr = metacard.getAttribute(Metacard.RESOURCE_DOWNLOAD_URL);
+        Attribute downloadUrlAttr = metacard.getAttribute(Core.RESOURCE_DOWNLOAD_URL);
         if (downloadUrlAttr != null) {
-            resourceUrl = (String) downloadUrlAttr.getValue();
+            Serializable downloadUrl = downloadUrlAttr.getValue();
+            if (downloadUrl instanceof String) {
+                resourceUrl = (String) downloadUrl;
+            }
         }
 
-        if (StringUtils.isNotBlank(resourceUrl)) {
+        if (StringUtils.isNotEmpty(resourceUrl)) {
             pathValueTracker.add(new Path(GmdConstants.LINKAGE_URI_PATH), resourceUrl);
         } else {
             URI resourceUri = metacard.getResourceURI();
             if (resourceUri != null) {
-
                 pathValueTracker.add(new Path(GmdConstants.LINKAGE_URI_PATH),
                         resourceUri.toASCIIString());
             }
         }
+    }
 
+    protected void addCRSInformation(MetacardImpl metacard,
+            XstreamPathValueTracker pathValueTracker) {
+        Attribute attribute = metacard.getAttribute(Location.COORDINATE_REFERENCE_SYSTEM_CODE);
+        if (attribute != null && CollectionUtils.isNotEmpty(attribute.getValues())) {
+            attribute.getValues()
+                    .forEach(serializable -> {
+                        if (serializable instanceof String) {
+                            String[] crsSplit = ((String) serializable).split(":");
```

Comment:
a nit, use the limit parameter on split just in case the CRS_CODE_PATH contains a colon? Not sure if a colon can even appear in the code path.